test/data.test.js

```diff
@@ -456,6 +456,28 @@ it("test_invalid_vr_length", () => {
     }
 });
 
+it("test_long_explicit_vr", () => {
```

Comment:
And here is a test for this, writing out about 200k to a single field in explicit vr.